Content.Shared/Lathe/LatheMessages.cs

```diff
@@ -9,11 +10,11 @@ public sealed class LatheUpdateState : BoundUserInterfaceState
 {
     public List<ProtoId<LatheRecipePrototype>> Recipes;
 
-    public List<LatheRecipePrototype> Queue;
+    public NetListAsArray<ProtoId<LatheRecipePrototype>> Queue;
 
```

Comment:
If you want to be sweaty make a deltastate for queue vs recipes and that will probably cut bandwidth by a large amount.